eng/pipelines/templates/steps/create-apireview.yml

```diff
@@ -0,0 +1,36 @@
+parameters:
+  ArtifactPath: $(Build.ArtifactStagingDirectory)
+  Artifacts: []
+  ConfigFileDir: $(Build.ArtifactStagingDirectory)/PackageInfo
+  Language: 'Python'
+  APIViewUri: 'https://apiviewstaging.azurewebsites.net/AutoReview/CreateApiReview'
```

Comment:
This should be updated to point to production by default

Reply:
👍 I noticed that you're not using the eng/common version. This will probably be defaulted in the eng/common version right?

Reply:
We will go live for python with this version first and then I will move this into eng/common.